sdk/dotnet/Pulumi/PublicAPI.Unshipped.txt

```diff
@@ -1,2 +0,0 @@
-Pulumi.ResourceOptions.ReplaceOnChanges.get -> System.Collections.Generic.List<string>
-Pulumi.ResourceOptions.ReplaceOnChanges.set -> void
```

Comment:
Unrelated changes that should have been previously moved from unshipped to shipped.

Reply:
I added my public api changes directly to shipped to avoid such artifacts.